modules/virt-importing-vm-wizard.adoc

```diff
@@ -63,6 +63,11 @@ ifdef::virt-importing-rhv-vm[]
 endif::[]
 ifdef::virt-importing-vmware-vm[]
 . Select a virtual machine or a template to import.
++
+[IMPORTANT]
+====
+The VMware VM name must not contain a forward slash (`/`). link:https://bugzilla.redhat.com/show_bug.cgi?id=1884012[(*BZ#1884012*)]
```

Comment:
Is it really necessary to reference the BZ in the docs? Typically I don't think we do, outside of release notes.

Reply:
I removed it.